### PR TITLE
Beam summit blogpost

### PR DESCRIPTION
**Pull request by matthiasa4**
Adding a blogpost to the website on the Beam Summit London aftermath, containing the presentations and recordings of the Summit.

### CODE REVIEW

**Comment by mxm** [comment]
Awesome. Thanks for contributing the blog post!

**Comment by matthiasa4** [comment]
Thanks for merging @tweise! And my pleasure @mxm :)

File: website/src/_posts/2018-10-30-beam-summit-aftermath.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+In the aftermath of the Summit, you can check the presentations of all the sessions.
+
+### Day 1: Use cases
+* [Day 1 - Session 1 - Large scale stream analytics with Apache Beam at Sky](https://drive.google.com/open?id=1hyHw7RVpFrFpli3vLt6JGBHrEm4BcgF-5nRdH1ZE8qo)
+* [Day 1 - Session 2 - Running Quantitative Analytics with Apache Beam](https://drive.google.com/open?id=1MxYrFDVoVFsrzbTtmr18zcbPFUU4nSdi)
+* [Day 1 - Session 3 - Talend Data Streams: Building Big Data pipelines with Apache Beam](https://drive.google.com/open?id=0B4bFLXEWuluSdVBJSnZrbTZjSGFHbnd4cExYOGZQU2hmY3lF)
+* [Day 1 - Session 4 - Lesson Learned from Migrating to Apache Beam for Geo-Data Visualisation](https://drive.google.com/open?id=1-GIUVn9QBtg6t-O8uINDkMO4PyZSU_HAEjMWuUHiYY4)
+
+### Day 2: Beginners track
+* [Day 2 - Beginner - Session 1 - Development Environment with Apache Beam](https://drive.google.com/open?id=1aFH6lhnVIq4Alu-_HItQ0QOddEPJQRqI5jV_t0o3CYI)
```
Comment (tweise) [inline website/src/_posts/2018-10-30-beam-summit-aftermath.md]: @matthiasa4 just noticed that links in this section all point to the same document. Can you please fix this?

Comment (matthiasa4) [reply website/src/_posts/2018-10-30-beam-summit-aftermath.md]: Oops! Thanks for spotting @tweise. Please find the right links in this PR: https://github.com/apache/beam/pull/6935

**Pull request merged**